NetworkPkg/HttpBootDxe/HttpBootClient.c

```diff
@@ -1097,6 +1128,55 @@ HttpBootGetBootFile (
     }
   }
 
+  //
+  // Add HTTP header field 5 (optional): Range
+  //
+  if (ResumingOperation) {
+    // Resuming a failed download. Prepare the HTTP Range Header
+    AsciiSPrint (
+      BaseAuthValue,

```

Comment:
Use a new variable for Range information. Reusing BaseAuthValue here is confusing.

Reply:
Ok.